sklearn/decomposition/nmf.py

```diff
@@ -842,8 +842,13 @@ class NMF(BaseEstimator, TransformerMixin):
     solver : 'pg' | 'cd'
         Numerical solver to use:
         'pg' is a (deprecated) Projected Gradient solver.
+
+        .. versionchanged:: 0.17
```

Comment:
Wednesday or Thursday.
I would be happy if this was done by tonight but I don't want to stress you. I can also work on the rest.